src/main/java/com/ericsson/ei/queryservice/ProcessAggregatedObject.java

```diff
@@ -0,0 +1,68 @@
+/***********************************************************************
```

Comment:
Please remove the ericsson internal headers and replace with the Ericsson Github headers in all files